docs/source/apidoc/core.rst

```diff
@@ -129,3 +144,8 @@ Sampler
 .. automodule:: pulser.sampler.samples
    :members:
 
+Result
+------------------
+.. automodule:: pulser.result
+   :members:
+   :show-inheritance:
```

Comment:
Here we can also include `QutipResult`, maybe. It's a bit of an awkward fit under "Core Features" but maybe it's not so bad, what do you think?

Reply:
Won't `QutipResult` be best in `simulation.rst` ?

Reply:
As we are presenting there `CoherentResults` and `NoisyResults`

Reply:
Yeah, it's a toss up to me, since it is a `Result` subclass. I'll let you choose :)